pom.xml

```diff
@@ -59,6 +52,10 @@
           <id>ome.maven</id>
           <url>http://artifacts.openmicroscopy.org/artifactory/maven/</url>
         </repository>
+        <repository>
```

Comment:
Hmmm..... then when others run into the same problem? `scc merge` & `depends-on`?